README.md

```diff
@@ -170,6 +171,9 @@ qjs >
 
 [swift integration]: https://github.com/AlwaysRightInstitute/SwiftyWasmer
 
+[zig logo]: https://raw.githubusercontent.com/ziglang/logo/master/zig-favicon.png
+[zig integration]: https://github.com/kubkon/wasmer-zig
```

Comment:
Why not using https://github.com/zigwasm/wasmer-zig?

Reply:
Ah, it's been probably moved recently.

Reply:
Yep, correct! We introduced `zigwasm` org only today. I'll push an update to Wasmer's README shortly.